tests/Integration/Subscriber/Optimization/DequeueJQueryMigrateSubscriber/dequeueJqueryMigrate.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace WP_Rocket\Tests\Integration\Subscriber\Optimization\Dequeue_JQuery_Migrate_Subscriber;
+
+use PHPUnit\Framework\TestCase;
+use Brain\Monkey;
+
+/**
+ * @covers WP_Rocket\Subscriber\Optimization\Dequeue_JQuery_Migrate_Subscriber::dequeue_jquery_migrate
+ *
+ * @group  jQueryMigrate
+ */
+class Test_DequeueJqueryMigrate extends TestCase {
+	private $orig_deps;
+	private $wp_scripts;
+	private $count;
+
+	public function setUp() {
+		parent::setUp();
+
+		global $wp_scripts;
```

Comment:
You can use `wp_scripts()` to get the instance instead of calling the global: https://developer.wordpress.org/reference/functions/wp_scripts/

With WP_Scripts, you also have access to a bunch of methods that could be useful for the tests instead of looking into the object properties: https://developer.wordpress.org/reference/classes/wp_scripts/

Reply:
Even better! Good catch @Tabrisrp

Reply:
Resolved 694a638e44eafd72823592e1476871a6caad14cb